app/controllers/amr_heatmap_controller.rb

```diff
@@ -54,6 +54,7 @@ def amr_counts
     render json: amr_data
   end
 
-  def amr_heatmap
+  def visualization
+    @samples = { sampleIds: params[:sampleIds] }
```

Comment:
Do `@sampleIds = params[:sampleIds]` here, and `{ sampleIds: JSON.parse('<%= raw escape_json(@sampleIds) %>') }` in the `.erb` file. This makes it more explicit in the `.erb` file exactly which values we're passing as props into `AMRHeatmapVis`.